src/i18n/locales/en/translation.json

```diff
@@ -331,7 +331,7 @@
       },
       "confirm_modal": {
         "title": "Create fidelity bond and time-lock funds?",
-        "body": "It will be impossible to spend time-locked funds until the fidelity bond expires. Date of expiration is {{date}}"
+        "body": "It will be impossible to spend time-locked funds until the fidelity bond expires {{ humanReadableDuration }}. Date of expiration is {{date}}."
```

Comment:
```suggestion
        "body": "It will be impossible to spend time-locked funds until the fidelity bond expires. Date of expiration is {{ humanReadableDuration }} on {{date}}."
```



Having this phrase at the beginning of a sentence, as opposed to at the end, gives it more emphasis. What do you think?

![image](https://user-images.githubusercontent.com/3358649/183053061-b136f706-2550-4797-a269-58ca242a9d25.png)